layouts/highlights.html

```diff
@@ -11,7 +11,7 @@
       <p>Check out a number of <a href="http://wiki.hl7.org/index.php?title=Open_Source_FHIR_implementations">Open Source FHIR<sup>®</sup> Implementations</a>.</p>
     </li>
     <li class="highlight-module">
-      <a href="/faq/"><span class="mega-octicon octicon-question"></span></a>
```

Comment:
this needs to be removed. We shouldn't be linking from the main page, since these are Millennium specific

Reply:
removed on 9bee62d